src/routes/relay/entities/relay.dto.entity.ts

```diff
@@ -0,0 +1,12 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export class RelayDto {
+  @ApiProperty()
+  to!: string;
+
+  @ApiProperty()
+  data!: string;
+
+  @ApiPropertyOptional({ type: BigInt, nullable: true })
+  gasLimit: string | undefined;
```

Comment:
For entities nullable properties we are using `${type} | null`. I'm not sure this could have a lot of implications, but maybe it's more consistent to stick to that instead of `${type} | undefined`, what do you think? Also a default value in the schema could be considered 🤔

Reply:
Changed in dc73902b4838bcb57f7a97a74f997cc572677a30.